packages/framework-integration-tests/package.json

```diff
@@ -13,25 +13,28 @@
     "@boostercloud/framework-core": "^0.15.1",
     "@boostercloud/framework-provider-aws": "^0.15.1",
     "@boostercloud/framework-provider-local": "^0.15.1",
-    "@boostercloud/framework-types": "^0.15.1"
+    "@boostercloud/framework-provider-kubernetes": "^0.15.1",
+    "@boostercloud/framework-types": "^0.15.1",
+    "aws-sdk": "2.764.0"
```

Comment:
@boostercloud/booster-core I had to move this `aws-sdk` package to dependencies because we are adding a second provider. This is a bug that @adriangmweb found recently.